ament_cmake_auto/cmake/ament_auto_package.cmake

```diff
@@ -87,6 +90,14 @@ macro(ament_auto_package)
     )
   endif()
 
+  # install folders to share
```

Comment:
"folders" -> "directories" (see https://stackoverflow.com/questions/5078676/what-is-the-difference-between-a-directory-and-a-folder).